boards/teensy31/include/board.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2017 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_teensy31 Teensy3.1 & 3.2
+ * @ingroup     boards
+ * @brief       Board specific files for Teensy3.1 & 3.2
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Teensy3.1 & 3.2 board
+ *
+ * @author      Loïc Dauphin <loic.dauphin@inria.fr>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "mtd.h"
+
+/* Use the on board RTC 32kHz clock for LPTMR clocking. */
+#undef LPTIMER_CLKSRC
+/** @brief Clock source for the LPTMR module */
+#define LPTIMER_CLKSRC LPTIMER_CLKSRC_ERCLK32K
+
+/** Disable hardware watchdog, for debugging purposes, don't use this on production builds. */
+#define DISABLE_WDOG    1
+
+/**
+ * @brief Use the UART1 for STDIO on this board
+ */
+#define UART_STDIO_DEV      UART_DEV(0)
+
+/**
+ * @brief   xtimer configuration
```

Comment:
Should be `@name`

Reply:
The error is also present for the [Mulle board](https://github.com/RIOT-OS/RIOT/blob/master/boards/mulle/include/board.h#L43).

Reply:
This should change soon, #8371